src/include/execution/compiler/operator/index_create_translator.h

```diff
@@ -0,0 +1,104 @@
+#pragma once
+#include <vector>
+#include "execution/compiler/expression/pr_filler.h"
+#include "execution/compiler/operator/operator_translator.h"
+#include "planner/plannodes/create_index_plan_node.h"
```

Comment:
Can this be forward declared? Not seeing its implementation used in the header.

Reply:
But then translator_factory.cpp will complain about inheritance of CreateIndexPlanNode

> Can this be forward declared? Not seeing its implementation used in the header.